modules/aql/arrow-query-language/build.gradle

```diff
@@ -1,6 +1,9 @@
 dependencies {
     compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
     compile project(':arrow-annotations')
+    compile project(':arrow-effects-extensions')
```

Comment:
I believe this should be the other way around, where the extension packages depend on AQL. Else this library will contain 3 different concurrency runtimes :D

Reply:
Okey dokey, I'll move those files to right packages